VueJS/index.js

```diff
@@ -4,10 +4,47 @@ import ListItem from './components/ListItem.js'
 new Vue({
   el: '#app',
   data: {
+    newTodoText: '',
+    counter:0,
+    editmode:[
+      false,false,false
+      ],
+    tempValues:[
+      { id: '0', message: null, },
+      { id: '1', message: null, },
+      { id: '2', message: null, }
+    ],
     todos: [
-      { id: '1', message: 'Foo', },
-      { id: '2', message: 'Bar', },
-      { id: '3', message: 'Baz', }
+      { id: '0', message: 'Foo', },
+      { id: '1', message: 'Bar', },
+      { id: '2', message: 'Baz', }
     ] // feel free to put the `todos` array where you find it most suitable
+    ,nextTodoId: 3
```

Comment:
Again this thing :disappointed: please avoid state that you need to keep updated. See https://github.com/m-oehme/Systems-Development-and-Frameworks/pull/2#discussion_r339841486 for a better solution. We talked about it in the seminar.